backend/hitas/services/thirty_year_regulation.py

```diff
@@ -466,11 +479,33 @@ def _determine_regulation_need(
         obfuscated_owners=[],
     )
 
+    if replacement_postal_codes:
+        missing: list[PostalCodeT] = [
+            postal_code
+            for postal_codes in replacement_postal_codes.values()
+            for postal_code in postal_codes
+            if postal_code not in price_by_area
+        ]
```

Comment:
A little more documentation on would be appreciated in general _(such as this, for example)_. 🙂

Reply:
IDK, this one is pretty self-documenting. Get a list of postal codes not in `price_by_area` (= "Aluekohtaiset keskineliöhinnat"), meaning we tried to use a postal code without price information and should raise an error. I think comments should be used pretty sparingly since you will have to maintain the comment as well.